app/mailers/candidate_mailer.rb

```diff
@@ -1,4 +1,6 @@
 class CandidateMailer < ApplicationMailer
+  layout 'candidate_email_with_footer'
```

Comment:
Does this need to be a tiny bit clearer?

```suggestion
  layout 'candidate_email_with_support_footer'
```

Reply:
Not even sure ‘footer’ is the right word here… but can’t think of a good alternative.